src/components/ProjectsPage.js

```diff
@@ -1,83 +1,117 @@
+                 {id: 2, title: "Project Title 3", company: "Organization Name 3"},
+                 {id: 3, title: "Project Title 4", company: "Organization Name 4"},
+                 {id: 4, title: "Project Title 5", company: "Organization Name 5"},
+                 {id: 5, title: "Project Title 6", company: "Organization Name 6"},
+                 {id: 6, title: "Project Title 7", company: "Organization Name 7"}]
 
-let companies = [{id: 0, title: "Project Title 1", name: "Organization Name 1"},
-                 {id: 1, title: "Project Title 2", name: "Organization Name 2"},
-                 {id: 2, title: "Project Title 3", name: "Organization Name 3"},
-                 {id: 3, title: "Project Title 4", name: "Organization Name 4"},
-                 {id: 4, title: "Project Title 5", name: "Organization Name 5"}]
 
 class ProjectsPage extends React.Component {
+  state = {
+    selected: -1
+  }
+
+  render() {
+    return (
+      <Fragment>
+        { this.state.selected === -1 &&
+        <Container className="pg-container">
+          <Row className="d-flex justify-content-center">
+            <Row className="d-flex justify-content-left">
+              {projects.map(project =>
+                <Col lg={4}>
+                  <Card className="broad card-btn" onClick={() => this.setState({selected: project.id})}>
+                    <Body>
+                      <Title> {project.title} </Title>
+                      <Subtitle className="mb-2 text-muted"> {project.company} </Subtitle>
+                      <Button className="urgent-tag"> Urgent </Button>
+                      <Text class="card-text">Project description. We are looking for a group of
+                                                   people to help us put together a simple website for
+                                                   people to email their state reps.
+                      </Text>
+                      <Link href="#" className="float-right" onClick={() => this.setState({selected: project.id})}>More details...</Link>
+                    </Body>
+                  </Card>
+                </Col>
+              )}
+            </Row>
+          </Row>
+        </Container>
+        }
+
+
+
+
+
+        { this.state.selected !== -1 &&
+        <Container className="pg-container center-div">
+          <Row>
+            <Col lg={4} className="overflow-auto scrolling project-col d-none d-lg-block">
+              {projects.map(project =>
+                <Card className = {`preview-card ${(this.state.selected !== project.id) ? "card-btn" : "selected-btn"}`}
+                      onClick={() => this.setState({selected: project.id})}>
+                  <Body>
+                    <Title> {project.title} </Title>
+                    <Subtitle className="mb-2 text-muted"> {project.company} </Subtitle>
+                    <Button className="urgent-tag"> Urgent </Button>
```

Comment:
could we make these tags non-hoverable and non-clickable as they are not buttons?

Reply:
yup